app-containers/containers-image/containers-image-5.29.2.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Default config and docs related to Containers' images"
+HOMEPAGE="https://github.com/containers/image"
+
+if [[ ${PV} == 9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/containers/image.git"
+else
+	SRC_URI="https://github.com/containers/image/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}/${P#containers-}"
+	KEYWORDS="~amd64 ~arm64 ~riscv"
+fi
+
+LICENSE="Apache-2.0"
+SLOT="0"
+
+BDEPEND=">=dev-go/go-md2man-2.0.3"
+PATCHES=(
+	"${FILESDIR}"/moving-policy-json-default-yaml.patch
+)
+
+src_compile() {
+	emake docs
+}
+
+src_install() {
+	emake DESTDIR="${ED}" install
+
+	insinto /etc/containers
+	doins registries.conf
+}
```

Comment:
The tests failed for this ebuild with network-sandbox, and also for the previous version in the same way, so maybe we should RESTRICT=test:
```
storage/storage_dest.go:33:2: github.com/containers/storage@v1.51.0: Get "https://proxy.golang.org/github.com/containers/storage/@v/v1.51.0.zip": dial tcp: lookup proxy.golang.org on [2600:8802:b00:eb8:c256:27ff:febf:e445]:53: dial udp [2600:8802:b00:eb8:c256:27ff:febf:e445]:53: connect: network is unreachable
storage/storage_transport.go:16:2: github.com/containers/storage@v1.51.0: Get "https://proxy.golang.org/github.com/containers/storage/@v/v1.51.0.zip": dial tcp: lookup proxy.golang.org on [2600:8802:b00:eb8:c256:27ff:febf:e445]:53: dial udp [2600:8802:b00:eb8:c256:27ff:febf:e445]:53: connect: network is unreachable
make: *** [Makefile:81: test] Error 1
 * ERROR: app-containers/containers-image-5.29.2::gentoo failed (test phase):
 *   Make test failed. See above for details.
```